romancal/datamodels/schemas/reference_files/flat.schema.yaml

```diff
@@ -0,0 +1,25 @@
+%YAML 1.1
+---
+id: "http://stsci.edu/schemas/roman_datamodel/reference_files/flat.schema"
```

Comment:
This schema uses ndarray validators like `ndim` and `datatype` so it needs the ASDF metaschema:

```yaml
$schema: http://stsci.edu/schemas/asdf/asdf-schema-1.0.0
```

Reply:
Done.